CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Not released
 
+- Add creatable functionality to Autocomplete [#828](https://github.com/CartoDB/carto-react/pull/828)
```

Comment:
You include also some Menu changes, right @vmilan ?

Reply:
You're right, I'll add it to the changelog